### ISSUE TITLE
Plain-ref mentions are not detected

### ISSUE THREAD

**Issue by clehner** [dominictarr/ssb-mentions#2]
A ssb refs in plain text is not detected as a mention:

``` js
ssb-mentions (master)@cel$ node
> require(".")('%A2LvseOYKDXyuSGlXl3Sz0F5j2khVCN6JTf8ORD/tM8=.sha256')
[]
```

It is being rendered as a link, as shown below, but it seems that the `extractor.link` function is not being called.

``` js
> require('ssb-marked')('%A2LvseOYKDXyuSGlXl3Sz0F5j2khVCN6JTf8ORD/tM8=.sha256')
'<p><a href="%A2LvseOYKDXyuSGlXl3Sz0F5j2khVCN6JTf8ORD/tM8=.sha256">%A2LvseO...</a></p>\n'
```

When the ref is a URL for a full link, it is detected:

``` js
> require(".")('[asdasd](%A2LvseOYKDXyuSGlXl3Sz0F5j2khVCN6JTf8ORD/tM8=.sha256)')
[ { link: '%A2LvseOYKDXyuSGlXl3Sz0F5j2khVCN6JTf8ORD/tM8=.sha256',
    rel: 'mentions',
    name: 'asdasd' } ]
```

I tried setting the `mentions: true` marked option, but that didn't seem to help.

**Comment by dominictarr** [comment]
I must have closed this by accident?

**Comment by dominictarr** [comment]
oh no, it was closed by merging your commit with closed #4 in it. That isn't rendered very clearly by github.

**Comment by clehner** [comment]
This is the issue that #4 fixed

Current behavior:

``` js
> require(".")('%A2LvseOYKDXyuSGlXl3Sz0F5j2khVCN6JTf8ORD/tM8=.sha256')
[ { link: '%A2LvseOYKDXyuSGlXl3Sz0F5j2khVCN6JTf8ORD/tM8=.sha256',
    rel: 'mentions',
    name: undefined } ]
```
